int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -338,10 +338,10 @@ for (var i = 0; i < buttonSelector.length; i++) {
 if (window.location.href.toLowerCase().match(/checkout/)) {
 	setEdqInputSelectors("shipping");
 } else {
-	setEdqInputSelectors();
+	setEdqInputSelectors("");
 }
```

Comment:
Generally it’s a better practice to set the default on the function level of this is just a default value you’re using here.

Reply:
Will change it to use setEdqInputSelectors(); doing a correction in the function

function setEdqInputSelectors(stageContentLocation) {
   stageContentLocation  = stageContentLocation  || "";
   //code